docs/apis/batch/libs/gelly.md

```diff
@@ -2250,14 +2250,33 @@ graph.run(new TranslateVertexValues(new LongValueAddOffset(vertexCount)));
     </tr>
 
     <tr>
-      <td>translate.<br/><strong>TranslateEdgeValues</strong></td>
+      <td>asm.translate.<br/><strong>TranslateEdgeValues</strong></td>
       <td>
         <p>Translate edge values using the given <code>TranslateFunction</code>.</p>
 {% highlight java %}
 graph.run(new TranslateEdgeValues(new Nullify()));
 {% endhighlight %}
       </td>
     </tr>
+
+    <tr>
+      <td>library.similarity.<br/><strong>JaccardIndex</strong></td>
+      <td>
+        <p>Measures the similarity between vertex neighborhoods. The Jaccard Index score  is computed as the number of shared numbers divided by the number of distinct neighbors. Scores range from 0.0 (no shared neighbors) to 1.0 (all neighbors are shared).</p>
```

Comment:
Why did you add this here and not in the "Usage" section of the library method?
I find it a bit confusing... You describe graph algorithms as building blocks for other algorithms. Does Jaccard index fall in this category?